lib/forms/find_support.yml

```diff
@@ -0,0 +1,113 @@
+_
+find_support:
+- key: are_you_out_of_employment
+  group: not_in_work
+  type: radio
+  required: true
+  options:
+    _ "yes"
```

Comment:
Are this options supposed to match the I18n options? Do the options also need to be defined here?

Reply:
Yeah, i was thinking these should be the equivalent to the `value` attribute on an HTML `<input>`. So the answer is completely abstracted from the content, but because checkboxes and radios are always a known list, we can use the values to then look up the appropriate text for each answer in the locale file.